src/app/auth/components/login/login.component.html

```
@@ -21,6 +21,15 @@ <h2 class="text-2xl md:text-3xl font-bold text-center mb-6">Login</h2>
           (formSubmit)="onSubmit($event)"
           class="w-full">
         </app-dynamic-form>
+        <div class="text-center mt-4">
+          <p class="text-sm text-gray-600">
+            Don't have an account? 
+            <a
+              (click)="goToRegister()"
+              class="text-blue-500 hover:underline cursor-pointer">
```

HTMLHint (1.5.0)

[warning] The id and class attribute values must be in lowercase and split by a dash.

(id-class-value)